demo.py

```diff
@@ -24,16 +27,33 @@
 your address bar to http://localhost:5000/oauth-callback?oauth_verifier=...etc
 """)
 
-consumer_key = input('Consumer key: ')
-consumer_secret = input('Consumer secret: ')
+try:
+    creds_doc = json.load(open("credentials.do_not_commit.json"))
+    consumer_key = creds_doc['consumer_key']
+    consumer_secret = creds_doc['consumer_secret']
+except:
```

Comment:
this could distinguish between 'file not found' and 'invalid json', but given that it's a demo script, I think this is also fine :-)